spec/Modules.tex

```diff
@@ -21,7 +22,11 @@ \section{Module Definitions}
 A module is declared with the following syntax:
 \begin{syntax}
 module-declaration-statement:
-  `module' module-identifier block-statement
+  privacy-specifier[OPT] `module' module-identifier block-statement
+
+privacy-specifier:
+  `private'
+  `public'
 
```

Comment:
I had thought the intention was for the syntax sections to be distinct and so saw no problem with the duplication.  I don't think someone should have to go searching for the definition of a rule if it is immediately relevant.